### DISCUSSION TITLE
Feature / pricing comparison Supabase Pro vs. TimescaleDB Cloud

### DISCUSSION THREAD

**Discussion by steffenstolze** [supabase/supabase#9116] [General]
Hey there!

I've recognised that you can enable TimescaleDB extension on new Supabase projects - awesome!

Checking the pricing for TimescaleDB Cloud (https://www.timescale.com/products#pricing), I was wondering: is a Supabase project with enabled TimescaleDB extension comparable (performance + features) with a TimescaleDB Cloud instance?

Both can be hosted on AWS. With TimescaleDB Cloud you can specifically choose a fixed storage, RAM and CPU config, while Supabase seems to provide a serverless, auto-scaling DBaaS.

Just for comparison:
- TimescaleDB Cloud with 100GB of storage will cost **~$560 a month**
- Supabase Pro would cost $25 base + $11.5 (92GB) + $4.5 (50GB egress) = **$41 a month**

Even if we'd consider 1.7TB "uncompressed data", Supabase Pro would result in $25 base + $211.5 (1692GB) + $148,5 (1650GB egress) = **$385 a month** - but if I understand it correctly, we should only assume 100GB, since TimescaleDB compresses the data.

So, is it safe to assume that Supabase Pro offers the same features and performance as TimescaleDB Cloud for almost **1/14** of the price, or do I miss something?

**Comment by GaryAustin1** [comment]
If you look on the Pricing page at Add-Ons
You should be able to get a feel for how the CPU/memory match up.
![image](https://user-images.githubusercontent.com/54564956/191854000-12b69c56-fdfc-495a-899e-ab486be5bd1b.png)

**Comment by steffenstolze** [comment]
Alright. So with Free and Pro you get a 2 vCPU / 1 GB configuration. I think I'd have to slightly correct my statement from above 

> while Supabase seems to provide a serverless, ~auto-scaling~ DBaaS

since it's not auto-scaling but bound to a specific configuration, right?

This is what the Supabase pricing page must mean with

> For production applications **with the option to scale**. 

"Option" meaning that I can choose to upgrade to a different configuration when needed.
Whether or not I have performance issues or a need so scale up has to be determined by myself. Hints on how that could be accomplished can be found in the Supabase docs [https://supabase.com/docs/guides/platform/performance](https://supabase.com/docs/guides/platform/performance).

Got it.

So comparing Supabase to TimescaleDB Cloud would mean that I'd need the Supabase XL Plan with 4 vCPU + 16 GB RAM, resulting in the above $41 + $200 for the XL plan = $241 compared to TimescaleDB Cloud $560.

Just out of curiosity: since Supabase [is using AWS EC2 T4g instances](https://aws.amazon.com/ec2/graviton/customers/) I assume that we're talking about a `t4g.xlarge` instance with 4 vCPU and 16 GB RAM?

How do they come up with the 4,750Mbps max. Disk IO

<img width="892" alt="image" src="https://user-images.githubusercontent.com/8490779/191920566-211ca9a8-e1e8-4b35-94a1-ac21a4ac0426.png">

when [EBS for this instance is capped](https://aws.amazon.com/ec2/instance-types/t4/) at 2,780Mbps

<img width="1177" alt="image" src="https://user-images.githubusercontent.com/8490779/191920813-8814b814-bfcb-40fb-acdc-80acfe1f03d3.png">

Or is the information regarding T4g outdated and are they using M6g under the hood?

<img width="833" alt="image" src="https://user-images.githubusercontent.com/8490779/191921640-8da183c7-307c-488a-afd2-11e8e9695126.png">

**Reply by shinebayar-g** [reply]
I think Supabase is integrated with Timescale Apache2 (Not TimescaleDB Community Edition) version which is [trimmed down](https://docs.timescale.com/timescaledb/latest/timescaledb-edition-comparison/) to like 30% of Timescale?
